routing/services/routing.yml

```diff
@@ -0,0 +1,72 @@
+services:
+    # default configuration for services in *this* file
+    _defaults:
+        public: false
+
+    routing.resolver:
+        class: Symfony\Component\Config\Loader\LoaderResolver
+
+    routing.loader.xml:
+        class: Symfony\Component\Routing\Loader\XmlFileLoader
+        arguments:
+            - '@file_locator'
+        tags: ['routing.loader']
```

Comment:
I assume this is not necessary if we're only allowing YAML config files, right?

Reply:
Why would you want to restrict this? Some module may have a need for flexible PHP config-files..
What we allow is set [here](https://github.com/simplesamlphp/simplesamlphp/pull/1178/files/2d7943ac54b97aad6efa77d182bd738b73d89bf4#diff-35d461a964eb96cfff3e6212ba17ade7R23)